client/scss/components/_is_preview.scss

```diff
@@ -0,0 +1,6 @@
+.is-preview {
+  [alt=''] {
+    background: repeating-linear-gradient(-45deg, color('orange-graphics'), color('orange-graphics') 10px, color('teal') 10px, color('teal') 20px);
+    padding: 50px;
+  }
+}
```

Comment:
Any chance of putting a text warning in so new editors know what this means?

Reply:
I couldn't think of a way to do it (`img`s don't have pseudo-elements, annoyingly). Do you have suggestions?